src/app/service/carpooling/carpooling.service.ts

```diff
@@ -105,4 +105,15 @@ export class CarpoolingService extends AbstractService implements CarpoolingServ
       }
     });
   }
+
+  getCarpoolings(token: string): Observable<any> {
+    return this.http.get<any>(`${environment.path}/TODO`, {
```

Comment:
```suggestion
    return this.http.get<any>(`${environment.path}/carpooling/booked`, {
```
Si c'est l'historique des co-voiturages reservé (https://github.com/DUT-Info-Montreuil/SAE-5.A-carmate-back/pull/44/files#diff-dca629db3e18fb8241ca7289732e44d4a600a8883df6da4409fbd417aa0782d3)